src/gdcdictionary/schemas/molecular_test.yaml

```diff
@@ -7101,6 +7250,19 @@ properties:
       - 26-50
       - 51-75
       - 76-100
```

Comment:
Are these meant to be deprecated going forward?

Reply:
@stilesj-uchicago Great q, this property actually captures a lot of different values from different questions on TCGA enrollment forms (https://www.nationwidechildrens.org/research/areas-of-research/biopathology-center/nci-ccg-project-team/the-cancer-genome-atlas/tcga-forms-and-documents) as well as other forms from other studies as well. It's sort of a catch all. 

As IMS is migrating clinical data from those forms, the values here will track the specific value ranges that were used for that specific study. So to answer, these ranges will probably not be deprecated, since we may not know the specific value for certain clinical data and where it would fall in a more specific range (e.g. would a value instance of 0-25 would be migrated to 0-10, 10-19 or 20-29?) We can probably bring up in data dict meeting to see how feasible this would be if we do have specific values if at all.